quarkus/panache/pom.xml

```diff
@@ -40,7 +40,7 @@
     </dependency>
     <dependency>
       <groupId>io.quarkus</groupId>
-      <artifactId>quarkus-resteasy</artifactId>
+      <artifactId>quarkus-resteasy-reactive</artifactId>
```

Comment:
You also need to update `quarkus-resteasy-jsonb` to `quarkus-resteasy-reactive-jsonb`